jenkins/e2e-image/Dockerfile-1.4

```diff
@@ -15,7 +15,7 @@
 # This file creates a build environment for building and running kubernetes
 # unit and integration tests
 
-FROM gcr.io/google-containers/kubekins-test:v20160822
+FROM gcr.io/k8s-testimages/kubekins-test:1.4-v20161007-49d47fe
```

Comment:
I am actually more towards combine e2e-images and test-images dir to avoid those duplications.